### PR TITLE
change: Show the source text of the source string as the content of t…

### PR DESCRIPTION
**Pull request by d29107d**
Directly show the source text of the source string as the content of the New source string event in the history.
Issue: https://github.com/WeblateOrg/weblate/issues/8350

## Checklist

- [ ] Lint and unit tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added documentation to describe my feature.
- [ ] I have squashed my commits into logic units.
- [ ] I have described the changes in the commit messages.

### CODE REVIEW

**Comment by d29107d** [comment]
@nijel 
Could you run the CI again or check why the checks failed?
I don't have permission to run CI again.

**Comment by nijel** [comment]
Did that now. There are some issues with the azure apt mirror (see https://github.com/actions/runner-images/issues/675).

**Comment by d29107d** [comment]
@nijel 
I contribute the code the first time and rebase the main branch, it needs a maintainer to approve running workflows.

File: weblate/trans/models/change.py [hunk 1]
```diff
@@ -592,6 +592,9 @@ def auto_status(self):
     def get_action_display(self):
         if self.action in self.PLURAL_ACTIONS:
             return self.PLURAL_ACTIONS[self.action] % self.plural_count
+        if self.action == self.ACTION_NEW_SOURCE:
+            return self.get_source()
```
Comment (nijel) [inline weblate/trans/models/change.py:596]: Display will be messed up with plural strings (that's also reason for failing tests). The value should be rendered using `format_translation` template tag in `weblate/templates/last-changes-content.html`.

Also, this is wrong place to display content, this method returns action name and is rendered in the left column while browsing changes.

**Pull request closed without merging**